org.kde.WaylandDecoration.QWhiteSurGtkDecorations.json

```
@@ -0,0 +1,40 @@
+{
+    "id": "org.kde.WaylandDecoration.QWhiteSurGtkDecorations",
+    "branch": "5.15-23.08",
+    "runtime": "org.kde.Platform",
+    "build-extension": true,
+    "sdk": "org.kde.Sdk",
+    "runtime-version": "5.15-23.08",
+    "appstream-compose": false,
+    "separate-locales": false,
+    "modules": [
+        {
+            "name": "qwhitesurgtkdecorations",
+            "buildsystem": "cmake-ninja",
+            "config-opts": ["-DCMAKE_INSTALL_PREFIX=/usr/share/runtime/lib/plugins/QWhiteSurGtkDecorations",
+                            "-DQT_PLUGINS_DIR=/usr/share/runtime/lib/plugins/QWhiteSurGtkDecorations"],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/FengZhongShaoNian/QWhiteSurGtkDecorations.git",
+                    "tag": "0.1.0"
+                }
+            ]
+        },
+        {
+            "name": "appdata",
+            "buildsystem": "simple",
+            "build-commands": [
+                "mkdir -p ${FLATPAK_DEST}/share/appdata",
+                "cp org.kde.WaylandDecoration.QWhiteSurGtkDecorations.appdata.xml ${FLATPAK_DEST}/share/appdata",
+                "appstream-compose --basename=org.kde.WaylandDecoration.QWhiteSurGtkDecorations --prefix=${FLATPAK_DEST} --origin=flatpak org.kde.WaylandDecoration.QWhiteSurGtkDecorations"
```

```suggestion
                "install -Dm0644 org.kde.WaylandDecoration.QWhiteSurGtkDecorations.appdata.xml ${FLATPAK_DEST}/share/appdata"
```

This should work automatically